barchart/schemas/migrate/tests/basic/expected.json

```diff
@@ -0,0 +1,10 @@
+{
+  "kind": "BarChart",
+  "spec": {
+    "calculation": "last-number",
+    "format": {
+      "decimalPlaces": 2,
+      "unit": "percent"
+    }
```

Comment:
Maybe you didn't finish your work, but here I find it is strange that the query is not migrated and the datasource setup as well